Mathlib/MeasureTheory/Measure/LevyProkhorovMetric.lean

```diff
@@ -233,6 +280,30 @@ noncomputable instance levyProkhorovDist_pseudoMetricSpace_probabilityMeasure :
 lemma LevyProkhorov.dist_def (μ ν : LevyProkhorov (ProbabilityMeasure Ω)) :
     dist μ ν = levyProkhorovDist μ.toMeasure ν.toMeasure := rfl
 
+/-- If `Ω` is a Borel space, then the Lévy-Prokhorov distance `levyProkhorovDist` makes
+`ProbabilityMeasure Ω` a metric space. The instance is recorded on the type synonym
+`LevyProkhorov (ProbabilityMeasure Ω) := ProbabilityMeasure Ω`.
+
+Note: For this metric to give the topology of convergence in distribution, one must
+furthermore assume that `Ω` is separable. -/
+noncomputable instance levyProkhorovDist_metricSpace_probabilityMeasure [BorelSpace Ω] :
+    MetricSpace (LevyProkhorov (ProbabilityMeasure Ω)) where
+  eq_of_dist_eq_zero := by
+    intro μ ν h
+    have same : μ.toProbabilityMeasure = ν.toProbabilityMeasure := by
```

Comment:
```suggestion
  show μ.toProbabilityMeasure = ν.toProbabilityMeasure
```
Then you can unindent and remove the `exact same` at the end.

Or better yet, you can use my `equiv` suggestion above and just make this line:
```suggestion
  apply LevyProkhorov.equiv.injective
```
and remove the defeq abuse in its entirety.